olm-catalog/serverless-operator/project.yaml

```diff
@@ -27,6 +27,8 @@ dependencies:
   net_istio: 1.3.0
   maistra: 2.1.0
 
+  serving_artifacts_branch: image_injection_1.3
+
```

Comment:
nit: maybe group this with `dependencies.serving` as put them closer to each other?